common-rendering/src/editorialPalette/background.ts

```diff
@@ -297,6 +373,107 @@ const headlineTag = (format: ArticleFormat): Colour => {
 	}
 };
 
+const relatedCard = (format: ArticleFormat): Colour => {
+
+	if (format.theme === ArticleSpecial.Labs) {
+		return neutral[93];
+	}
+
+	switch (format.design) {
+		case ArticleDesign.LiveBlog:
+			switch (format.theme) {
+				case ArticlePillar.News:
+					return news[200];
+				case ArticlePillar.Lifestyle:
+					return lifestyle[200];
+				case ArticlePillar.Sport:
+					return sport[200];
+				case ArticlePillar.Culture:
+					return culture[200];
+				case ArticlePillar.Opinion:
+					return opinion[200];
+				case ArticleSpecial.SpecialReport:
+					return specialReport[200];
+			}
+		case ArticleDesign.Audio:
+		case ArticleDesign.Video:
+		case ArticleDesign.Gallery:
+			return neutral[10];
+	}
```

Comment:
Can I recommend adding a `default` to any inner nested `switch` statements in this file (I'd recommend using news as the default)? The `switch` statement in JS can cause some subtle bugs due to its fall-through behaviour.

I'll try to demonstrate what I mean by using the above code as an example. Within the `LiveBlog` case there's currently no guarantee that every `theme` has been handled. At the moment we can figure out by inspection that they have, because six of the seven are handled here and labs is handled in the `if` statement above. But what if someone deletes that `if` statement or we add an extra type to `theme`? Your expectation might be that if none of the themes match then both switches will be broken out of and we'll hit `return neutral[97]` at the bottom of the function. But what actually happens is that there's no early `return` in the `LiveBlog` case, so JS _falls through_ to the cases below _without checking_ that they match... and we hit `return neutral[10]`.

More details about this behaviour [on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch#what_happens_if_i_forgot_a_break). The implication there is that a `break` is the way to solve this, but I'd recommend a `default` return here because that's more consistent with the pattern-matching-style approach we take to `switch` statements in this codebase.

If this code were in the AR part of the codebase I think the linter would have caught this 🤔. @jamesgorrie possible motivation here for splitting up common-rendering: the tooling here isn't as robust as in the AR/DCR projects?